docs/source/array-api.rst

```diff
@@ -127,6 +127,7 @@ Top level user functions:
    tan
    tanh
    tensordot
+   tile
```

Comment:
Forgot to add the `autofunction` part. Added in PR ( https://github.com/dask/dask/pull/2155 ).